Makefile

```
@@ -0,0 +1,35 @@
+.PHONY: dev-venv test snapshot history playback clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)